node/sc/bridgepool/sorted_map_list.go

```diff
@@ -84,6 +84,16 @@ func (m *ItemSortedMap) Get(nonce uint64) itemWithNonce {
 	return m.items[nonce]
 }
 
+// Exist returns if the nonce exist.
+func (m *ItemSortedMap) Exist(nonce uint64) bool {
+	m.mu.Lock()
```

Comment:
How about `RLock()`?

Reply:
It's not RW Mu in the sorted map. I will consider it later.